auth/shared-signal-receiver/common/http/sendHttpRequest.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+/* eslint-disable @typescript-eslint/naming-convention */
+
+/**
+ *
+ * @param delayMillis
+ */
+async function wait(delayMillis: number): Promise<void> {
+  // eslint-disable-next-line promise/avoid-new
+  await new Promise((resolve) => setTimeout(resolve, delayMillis));
+}
+
+/**
+ *
+ * @param request
+ * @param attempt
+ * @param baseDelayMillis
+ */
+async function retryWithExponentialBackoffAndFullJitter(
```

Comment:
Used from DI approach [link](https://github.com/govuk-one-login/mobile-platform-back/blob/main/backend-api/src/common/http/sendHttpRequest.ts) - could alternatively be created using axios and axios-retry.